claude-ops/scripts/macos/pid-performance-monitor.py

```
@@ -0,0 +1,380 @@
+    return subprocess.run(["taskpolicy", "-b", "-p", str(pid)], capture_output=True).returncode == 0
+
+
+def renice_background(pid: int, current_nice: int) -> bool:
+    if current_nice >= 10:
+        return True
+    return subprocess.run(["renice", "10", "-p", str(pid)], capture_output=True).returncode == 0
+
+
+def health_score(cpu: dict[str, float], running: int, stuck: int, mem: int, swap: float) -> int:
+    score = 100
+    if cpu["idle"] < 20: score -= 25
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)